helmfile/values/fluentd-configmap.yaml.gotmpl

```diff
@@ -65,7 +65,7 @@ aggregator:
       </match>
     {{ if eq .Values.objectStorage.type "gcs" -}}
     gcp_credentials.json: |
-      {{ .Values.objectStorage.gcs.keyfileData | nindent 4 }}
+      {{ .Values.objectStorage.gcs.keyfileData | nindent 8 }}
```

Comment:
Maybe?
```suggestion
      {{ .Values.objectStorage.gcs.keyfileData | indent 6 }}
```

Reply:
I thought I remembered some reason why it had to be 8 rather than 6, but I tried it with 6 now and it works so it would make sense to change that. Using `indent` instead of `nindent` doesn't work for me though, I'm not sure what the difference is between the two?

Reply:
Okay, if it works it's fine with me.
`nindent` prepends a newline to the string, http://masterminds.github.io/sprig/strings.html

Reply:
If you use `indent` it will use your current indentation plus add indentation for the first line. Therefore I think we should use `nindent` to make sure that the whole block is using the same indentation.

But `nindent 6` should be enough, but then again 8 should work as well. (not 4 though)